sources/packages/forms/src/form-definitions/educationprogram.json

```diff
@@ -1,5871 +1,6389 @@
 {\r\n        cipDigit: \"52\",\r\n        sabcCode: \"Z\"\r\n    },\r\n    {\r\n        cipDigit: \"01\",\r\n        sabcCode: \"Z\"\r\n    },\r\n    {\r\n        cipDigit: \"03\",\r\n        sabcCode: \"Z\"\r\n    },\r\n    {\r\n        cipDigit: \"05\",\r\n        sabcCode: \"X\"\r\n    },\r\n    {\r\n        cipDigit: \"16\",\r\n        sabcCode: \"X\"\r\n    },\r\n    {\r\n        cipDigit: \"23\",\r\n        sabcCode: \"X\"\r\n    },\r\n    {\r\n        cipDigit: \"24\",\r\n        sabcCode: \"X\"\r\n    },\r\n    {\r\n        cipDigit: \"25\",\r\n        sabcCode: \"X\"\r\n    },\r\n    {\r\n        cipDigit: \"42\",\r\n        sabcCode: \"X\"\r\n    },\r\n    {\r\n        cipDigit: \"27\",\r\n        sabcCode: \"X\"\r\n    },\r\n    {\r\n        cipDigit: \"30\",\r\n        sabcCode: \"X\"\r\n    },\r\n    {\r\n        cipDigit: \"38\",\r\n        sabcCode: \"X\"\r\n    },\r\n    {\r\n        cipDigit: \"40\",\r\n        sabcCode: \"X\"\r\n    },\r\n    {\r\n        cipDigit: \"45\",\r\n        sabcCode: \"X\"\r\n    },\r\n    {\r\n        cipDigit: \"50\",\r\n        sabcCode: \"X\"\r\n    },\r\n    {\r\n        cipDigit: \"54\",\r\n        sabcCode: \"X\"\r\n    },\r\n    {\r\n        cipDigit: \"55\",\r\n        sabcCode: \"X\"\r\n    },\r\n    {\r\n        cipDigit: \"13\",\r\n        sabcCode: \"W\"\r\n    },\r\n    {\r\n        cipDigit: \"19\",\r\n        sabcCode: \"W\"\r\n    },\r\n    {\r\n        cipDigit: \"28\",\r\n        sabcCode: \"W\"\r\n    },\r\n    {\r\n        cipDigit: \"31\",\r\n        sabcCode: \"W\"\r\n    },\r\n    {\r\n        cipDigit: \"44\",\r\n        sabcCode: \"W\"\r\n    },\r\n    {\r\n        cipDigit: \"60\",\r\n        sabcCode: \"V\"\r\n    },\r\n    {\r\n        cipDigit: \"04\",\r\n        sabcCode: \"U\"\r\n    },\r\n    {\r\n        cipDigit: \"10\",\r\n        sabcCode: \"U\"\r\n    },\r\n    {\r\n        cipDigit: \"14\",\r\n        sabcCode: \"U\"\r\n    },\r\n    {\r\n        cipDigit: \"15\",\r\n        sabcCode: \"U\"\r\n    },\r\n    {\r\n        cipDigit: \"29\",\r\n        sabcCode: \"U\"\r\n    },\r\n    {\r\n        cipDigit: \"41\",\r\n        sabcCode: \"U\"\r\n    },\r\n    {\r\n        cipDigit: \"26\",\r\n        sabcCode: \"U\"\r\n    },\r\n    {\r\n        cipDigit: \"22\",\r\n        sabcCode: \"S\"\r\n    },\r\n    {\r\n        cipDigit: \"43\",\r\n        sabcCode: \"S\"\r\n    },\r\n    {\r\n        cipDigit: \"51\",\r\n        sabcCode: \"R\"\r\n    },\r\n    {\r\n        cipDigit: \"39\",\r\n        sabcCode: \"Q\"\r\n    },\r\n    {\r\n        cipDigit: \"12\",\r\n        sabcCode: \"P\"\r\n    },\r\n    {\r\n        cipDigit: \"21\",\r\n        sabcCode: \"P\"\r\n    },\r\n    {\r\n        cipDigit: \"46\",\r\n        sabcCode: \"P\"\r\n    },\r\n    {\r\n        cipDigit: \"47\",\r\n        sabcCode: \"P\"\r\n    },\r\n    {\r\n        cipDigit: \"48\",\r\n        sabcCode: \"P\"\r\n    },\r\n    {\r\n        cipDigit: \"49\",\r\n        sabcCode: \"P\"\r\n    }\r\n];",
```

Comment:
Can we have this similar to level of studycodes, like a key value pair? where the key is the cipDigit and the value as sabcCode. SO that processing can be simplified.

Reply:
Updated.